core/vm/jump_table.go

```diff
@@ -211,8 +211,22 @@ func instructionSetForConfig(config ctypes.ChainConfigurator, isPostMerge bool,
 	}
 
 	// Cancun
+	/*
+		func newCancunInstructionSet() JumpTable {
+			instructionSet := newShanghaiInstructionSet()
+			enable4844(&instructionSet) // EIP-4844 (BLOBHASH opcode)
+			enable7516(&instructionSet) // EIP-7516 (BLOBBASEFEE opcode)
+			enable1153(&instructionSet) // EIP-1153 "Transient Storage"
+			enable5656(&instructionSet) // EIP-5656 (MCOPY opcode)
+			enable6780(&instructionSet) // EIP-6780 SELFDESTRUCT only in same transaction
+			return validate(instructionSet)
+		}
+	*/
 	if config.IsEnabledByTime(config.GetEIP4844TransitionTime, bt) {
```

Comment:
Shall we already add enablement using block number for ETC? Or we prefer doing this before hard-forks for needed EIPs only?

Reply:
I'd say its a TODO. It seems unfinished without them.

Reply:
Fixed with [de0e288](https://github.com/etclabscore/core-geth/pull/575/commits/de0e2883e9d6e3af1a56fb239d5c40ee95f6e8b4)